src/Tickets/Admin/Upsell.php

```diff
@@ -78,23 +80,10 @@ public function show_on_attendees_page() {
 			return;
 		}
 
-		$has_tickets_plus = class_exists( '\Tribe__Tickets_Plus__Main', false );
-		$has_wallet_plus  = class_exists( '\TEC\Tickets_Wallet_Plus\Plugin', false );
+		$has_tickets_plus = did_action( 'tec_container_registered_provider_Tribe__Tickets_Plus__Service_Provider' );
```

Comment:
We should ticket adding a "tec-tickets-plus-loaded" hook if we need to go this route.